src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+spring.jpa.hibernate.ddl-auto=update
+spring.datasource.url=jdbc:mysql://localhost:3306/online_delivery
+spring.datasource.username=springuser
```

Comment:
Convert this file to yaml format so it is easier to read. 
Also sensitive data like url, username and password should be stored in env variables, have default value and be referred in the yaml file (e.g. `${URL:jdbc:mysql://localhost:3306/online_delivery}`

Reply:
Done